addons/mail/static/src/core/common/message.xml

```diff
@@ -93,8 +93,19 @@
                                                 <Composer t-if="state.isEditing" autofocus="true" composer="message.composer" messageComponent="constructor" onDiscardCallback.bind="exitEditMode" onPostCallback.bind="exitEditMode" mode="env.inChatter ? 'extended' : 'compact'" sidebar="false"/>
                                                 <t t-else="">
                                                     <em t-if="message.subject and !message.isSubjectSimilarToOriginThreadName and !message.isSubjectDefault" class="mb-1 me-2">Subject: <t t-esc="message.subject"/></em>
-                                                    <t t-if="message.body" t-out="message.body"/>
-                                                    <small t-if="message.editDate" class="o-mail-Message-edited fst-italic text-muted" t-att-class="{ 'ms-2': !message.isBodyEmpty }" t-att-title="message.editDatetimeHuge">(edited)</small>
+                                                    <t t-if="message.body">
+                                                        <t t-if="message.displayTranslation" t-out="message.translatedBody"/>
+                                                        <t t-else="" t-out="message.body"/>
+                                                    </t>
```

Comment:
Suggestion:
```js
<t t-if="message.translatedBody" t-out="message.translatedBody"/>
<t t-elif="message.body" t-out="message.body"/>
```
A bit more idiomatic, no need to indent and nest conditionals. Less LOCs :)

Reply:
Looks much better indeed! Updated my code.